### DISCUSSION TITLE
Setting suggestion for indexing Live Photos not as Video

### DISCUSSION THREAD

**Discussion by t4nec** [photoprism/photoprism#4582] [Q&A]
Hello everyone,

First of all, thank you for developing such a great product. I'm very excited about it and have been experimenting with it over the last few days.

I’ve searched through GitHub issues and Reddit but haven’t found a 100% answer to my question.

I’m a big fan of Live Photos on iOS (iPhone 12 Pro, iOS 17.6.1) and was excited to see that PhotoPrism supports them. However, the issue I’m facing is that most of my Live Photos are being identified as videos, which creates a mess in the Video section and makes it very difficult to find actual videos. All of my media was imported using PhotoSync Premium (another great app from my initial impression).

I’m planning to create a new instance and reindex all of my photos to hopefully make Live Photos appear correctly, but I would really appreciate any suggestions on how to configure the Library index.

All of the files have the same naming pattern as in the example provided. I also looked into the details of the stacked files, and while they are stacked, only a few are identified as Live Photos (this specific example is not marked as one).

![image](https://github.com/user-attachments/assets/0b2771fd-6121-4ffb-8618-65af4216fee9)

Are there any settings that can be applied for better detection of Live Photos? I’m also considering removing all MOV files that support static pictures to resolve the issue, but if it’s possible to have this working correctly, that would be ideal and would motivate me even more to purchase a license.

**Comment by t4nec** [comment]
Still digging into this I have noticed that most of the picture, not identified as Live Photo there is a mismatch between the orientation of picture and orientation of MOV files. 

![image](https://github.com/user-attachments/assets/17bd018b-10f0-4f5c-a00f-eb367e52b55c)
![image](https://github.com/user-attachments/assets/ac5ce926-4ff4-4d40-b804-49966b6845d5)


In the successful Live Photo's, both picture and video files have same orientation, detected by photoprism.  

Edit 1:

On the photos not identified as Live Photo, there is no duration of the video (MOV) file, while on the Live photo there is Duration and few more details.

**NO LIVE PHOTO**
![image](https://github.com/user-attachments/assets/26022cb1-9c9c-4204-ba13-b11a31eccbfc)

**LIVE PHOTO:**
![image](https://github.com/user-attachments/assets/719eca5d-4296-41ca-a284-34e167fb467c)

**Comment by graciousgrey** [comment]
Hello,

Are you running the latest version of our docker image? Do you use hardware transcoding?

If no duration can be detected maybe there is something wrong with the metadata of the files.

**Reply by t4nec** [reply]
Yes, latest release, not using hardware transcoding. I'm running Docker in VM on top of PROXMOX. Do you think passthrough of iGPU of the host to the container may fix the issue in rebuilding the instance ?

**Reply by graciousgrey** [reply]
Can you reproduce the issue on our [public demo](https://demo.photoprism.app/library/browse)? If not, this would be a hint that the the problem is related to your setup.

**Reply by t4nec** [reply]
Of course, but cannot find login details, so I can setup PhotoSync to upload to public instance.

Edit:
Synced the PhotoSync to computer and the uploaded to demo provided and everything looks fine with picture, which where not indexed as Live Photo in my instance.

Meanwhile I have made a second instance for my wife and for now everything looks fine.

I will try to delete existing instance for my photos and create a new one and will report the results after that.

**Comment by t4nec** [comment]
Update on the issue. After clear redeploying of instance everything looks a lot better, but still some images not detected as live photo. Common thing between them that I have noticed is their duration is 4 seconds instead of 3, which may case the inconvenience. 

I don't know how many people is experiencing this issue, but maybe it will be good to have an option to manually review some of the items and mark the as Live Photos.

**Reply by lastzero** [reply]
You can manually change the picture type in the edit dialog:

![Screenshot 2024-10-21 at 16 01 45](https://github.com/user-attachments/assets/67d5f9f1-0a15-4786-9473-7fc5e0cf2ed2)

**Reply by t4nec** [reply]
This is great! Is there a way to change their type in batches?

**Reply by lastzero** [reply]
Not through the user interface at this time.
